app/schemas/base.py

```
@@ -0,0 +1,35 @@
+
+    class Config:
+        from_attributes = True
+        json_encoders = {
+            datetime: lambda dt: (
+                dt.astimezone(timezone.utc).isoformat().replace('+00:00', 'Z')
+                if dt and dt.tzinfo
+                else dt.replace(tzinfo=timezone.utc).isoformat().replace('+00:00', 'Z')
+                if dt
+                else None
+            )
+        }
```

Ruff (0.14.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)